### PR TITLE
edit front-end file structure

### PR DESCRIPTION
**Pull request by MohammedAlghazali**
relates #54

### CODE REVIEW

File: server/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const {
+  generalSchema,
+  cohortSchema,
+  projectSchema,
+  editCohortSchema,
+  studentSchema,
+  loginSchema,
+} = require('./validation');
+
+module.exports = {
```
Comment (tomduggan85) [inline server/utils/index.js:10]: @MohammedAlghazali This seems good right now.  In the future, if you have a lot of other kinds of utils (things that are not schemas) you could organize this further by putting all of these schemas into a named export:
```
module.exports = {
  schemas: {
    generalSchema,
    cohortSchema,
    projectSchema,
    ...
  },
  otherUtilExport,
  ...
};
```

**Comment by tomduggan85** [comment]
@MohammedAlghazali Nice reorganization / refactor! This looks good to me.  It's good to treat these kinds of changes as high-priority and get them merged quickly, since they affect all of the other developers on the team 👍

File: server/utils/index.js (hunk 1 above)
Comment (MohammedAlghazali) [reply server/utils/index.js:10]: yes, it will be very important to do that

**Comment by MohammedAlghazali** [comment]
@tomduggan85 thank you very much :grinning:

**Pull request merged**